src/functionalTest/java/uk/gov/hmcts/reform/hmi/people/PeopleTest.java

```diff
@@ -35,6 +36,7 @@ void setup() {
     }
 
     @Test
+    @Disabled
```

Comment:
Are these two tests correct to be disabled and if so could we add a comment detailing why?